app/views/overview/_list-row-actions.html

```diff
@@ -25,7 +25,7 @@
             </li>
             <!-- FIXME: Can't enable canI checks on svc cat resources until we have aggregation
             <li ng-if="(row.state.serviceInstances | hashSize) > 0 && {resource: 'bindings', group: 'servicecatalog.k8s.io'} | canI : 'create'" role="menuitem"> -->
-            <li ng-if="('pod_presets' | enableTechPreviewFeature) && (row.state.serviceInstances | hashSize) > 0" role="menuitem">
+            <li ng-if="('pod_presets' | enableTechPreviewFeature) && (row.state.bindableServiceInstances | hashSize) > 0" role="menuitem">
```

Comment:
bindableServiceInstances should be an array now, since you used _.filter, so you can just check row.state.bindableServiceInstances.length